include/wx/qt/radiobut.h

```diff
@@ -42,6 +47,8 @@ class WXDLLIMPEXP_CORE wxRadioButton : public wxControl
 private:
     QRadioButton *m_qtRadioButton;
 
+    static std::map<wxWindow*, QButtonGroup*> m_lastGroup;
```

Comment:
I'd make this a global variable or, even better, a static variable inside a function, in `radiobut.cpp` instead. This would hide this implementation dependency where it belongs.

Also, by wrapping it in a function, i.e. doing
```
typedef std::map<wxWindow*, QButtonGroup*> WindowToButtonGroupMap;
static WindowToButtonGroupMap& GetWindowToButtonGroupMap()
{
    static WindowToButtonGroupMap s_map;
    return s_map;
}
```
we avoid initializing this map until it's really needed, which means avoiding a tiny delay on application startup (and while this is almost certainly unmeasurable on its own, such tiny delays tend to add up to something noticeable with time).

Reply:
Noted. Will implement this.